sdk/lib/atl/atlwin.h

```diff
@@ -587,20 +587,31 @@ class CWindow
     }
 
 #ifdef __ATLSTR_H__
-    UINT GetDlgItemText(int nID, CSimpleString& string)
+    UINT GetDlgItemText(int nID, CSimpleString& strText) const
     {
         HWND item = GetDlgItem(nID);
+        if (!item)
+        {
+            strText.Empty();
+            return 0;
+        }
+#if 1
```

Comment:
Depending on what you'll tell me (either keep original code, or use instead the single line from below) I will adjust that.